public/assets/styles/index.scss

```diff
@@ -0,0 +1,51 @@
+@import url(https://fonts.googleapis.com/css?family=Zilla+Slab:300,300i,400,500,600,700%7COpen+Sans:300,400,600,700,800);
+@import './variables';
+@import './login';
```

Comment:
why have you removed the mixins import in this file? Please put it back in so that it does not affect others who are using mixins. Also, include a new line after line 1 for clarity. Thanks.

Reply:
Cool. The latest pull I had from develop never had that mixins file. My PR doesn't make use of any mixins file. So whosoever is to rebase after any PR gets merged can bring all needed files together.

Reply:
For the line 1, let me add that.

Reply:
We sent the same index.scss file to everyone to use via slack on valinor-backend general. This file is not based on develop. You are the only one who removed it. Whether you are using it or not, it should be in there. Please put it back in.